<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FaceDetectForm</class>
 <widget class="QWidget" name="FaceDetectForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>625</width>
    <height>464</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_5">
   <property name="spacing">
    <number>2</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QSplitter" name="splitter_3">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="handleWidth">
      <number>1</number>
     </property>
     <widget class="QSplitter" name="splitter_2">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QGroupBox" name="PlayerGroupBox">
       <property name="title">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="margin">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="PreviewLabel">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="pixmap">
           <pixmap resource="../IDE.qrc">:/Page/rc/Page/VideoScreen.png</pixmap>
          </property>
          <property name="scaledContents">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <property name="sizeConstraint">
           <enum>QLayout::SetDefaultConstraint</enum>
          </property>
          <item>
           <widget class="QPushButton" name="BtnOpenCamera">
            <property name="text">
             <string>OpenCamera</string>
            </property>
            <property name="icon">
             <iconset resource="../IDE.qrc">
              <normaloff>:/Menu/rc/Menu/notpass.png</normaloff>:/Menu/rc/Menu/notpass.png</iconset>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="BtnTakePicture">
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="../IDE.qrc">
              <normaloff>:/Menu/rc/Menu/camera.png</normaloff>:/Menu/rc/Menu/camera.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>25</width>
              <height>25</height>
             </size>
            </property>
            <property name="flat">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QCheckBox" name="RealTimeDetectSelect">
            <property name="text">
             <string>RealTime</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QGroupBox" name="PictureQueneGroupBox">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>150</height>
        </size>
       </property>
       <property name="title">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="margin">
         <number>0</number>
        </property>
        <item>
         <widget class="QListWidget" name="PictureQueneList"/>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <property name="spacing">
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>人脸识别文件</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="CompleterLineEdit" name="DetectConfEdit"/>
          </item>
          <item>
           <widget class="QToolButton" name="DetectConfSelect">
            <property name="text">
             <string>...</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="BtnGoto">
            <property name="maximumSize">
             <size>
              <width>22</width>
              <height>22</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="../IDE.qrc">
              <normaloff>:/Menu/rc/Menu/next.png</normaloff>:/Menu/rc/Menu/next.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>20</width>
              <height>20</height>
             </size>
            </property>
            <property name="flat">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QGroupBox" name="MeidaListGroupBox">
       <property name="title">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="margin">
         <number>0</number>
        </property>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <property name="spacing">
           <number>2</number>
          </property>
          <item>
           <widget class="QComboBox" name="MediaType">
            <property name="maximumSize">
             <size>
              <width>150</width>
              <height>16777215</height>
             </size>
            </property>
            <item>
             <property name="text">
              <string>All</string>
             </property>
             <property name="icon">
              <iconset resource="../IDE.qrc">
               <normaloff>:/Page/rc/Page/media.png</normaloff>:/Page/rc/Page/media.png</iconset>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Camera</string>
             </property>
             <property name="icon">
              <iconset resource="../IDE.qrc">
               <normaloff>:/Menu/rc/Menu/eye.png</normaloff>:/Menu/rc/Menu/eye.png</iconset>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Video</string>
             </property>
             <property name="icon">
              <iconset resource="../IDE.qrc">
               <normaloff>:/Page/rc/Page/mp4.png</normaloff>:/Page/rc/Page/mp4.png</iconset>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Audio</string>
             </property>
             <property name="icon">
              <iconset resource="../IDE.qrc">
               <normaloff>:/Page/rc/Page/mp3.png</normaloff>:/Page/rc/Page/mp3.png</iconset>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Image</string>
             </property>
             <property name="icon">
              <iconset resource="../IDE.qrc">
               <normaloff>:/Page/rc/Page/jpg.png</normaloff>:/Page/rc/Page/jpg.png</iconset>
             </property>
            </item>
           </widget>
          </item>
          <item>
           <widget class="CompleterLineEdit" name="MediaPath"/>
          </item>
          <item>
           <widget class="QPushButton" name="MediaPathSelect">
            <property name="minimumSize">
             <size>
              <width>25</width>
              <height>20</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>25</width>
              <height>20</height>
             </size>
            </property>
            <property name="text">
             <string>...</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTreeWidget" name="MediaTreeList">
          <column>
           <property name="text">
            <string notr="true">Media Source</string>
           </property>
           <property name="icon">
            <iconset resource="../IDE.qrc">
             <normaloff>:/Menu/rc/Menu/StreamMedia.png</normaloff>:/Menu/rc/Menu/StreamMedia.png</iconset>
           </property>
          </column>
          <item>
           <property name="text">
            <string>LocalDisk</string>
           </property>
           <property name="icon">
            <iconset resource="../IDE.qrc">
             <normaloff>:/Page/rc/Page/rc.png</normaloff>:/Page/rc/Page/rc.png</iconset>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CompleterLineEdit</class>
   <extends>QLineEdit</extends>
   <header location="global">completertool.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../IDE.qrc"/>
 </resources>
 <connections/>
</ui>
